tests/bench/run_tests.py

```
@@ -0,0 +1,697 @@
+    ) -> bool:
+        """
+        Wait for the deriver queue to be empty.
+
+        Args:
+            honcho_client: Honcho client instance
+            timeout: Maximum time to wait in seconds
+
+        Returns:
+            True if queue is empty, False if timeout exceeded
+        """
+        time.sleep(1)
```

Ruff (0.12.2)

Async functions should not call `time.sleep`

(ASYNC251)